README.md

```diff
@@ -1,2 +1,206 @@
+
+Probably the most striking difference between them, is that SQLite is a file-based database. When you query SQLite, it manipulates and fetches data by making direct calls to a file holding the data. That file, essentially, *is* the SQLite database. Postgres works very differently: it fetches data through an interface of ports and sockets that connect to a database server that must be installed in the environment in which the application is running.
+
+The fact that SQLite is just a file makes it an insanely lightweight and fast solution. So SQLite is a great technology to use when what is needed is speed and portability, and when the application in question is relatively simple. When, however, an application is larger; when, in particular, multiple parties (or users) may need to access the database at the same time, then SQLite begins to show its limitations. Because the SQLite database is just a file there's now mechanism for maintaining multiple database users who may be accessing the database at the same time. In SQLite there's just one user and only one write process can take place at one time. For many large-scale applications, this limitation is simply prohibitive. In this scenario, many software engineers will turn to a full-fledged relational database like Postgres. 
+
+## Installing PostgreSQL
+ 
+Now that we have an idea about the characteristics of Postgres, let's get it installed. Yes it needs to be installed! Here, of course, we encounter one of the limitations of Postgres compared to SQLite. It's definitely a bit more of an involved process to get it going on our systems.  Nonetheless, we can do it!
+
+One of the reason that install Postgres can be complex is that it depends on your environment. Setup and configuration, in general, is often frustrating because of the idiosyncacies of each environment. But we'll go through some methods here that should be pretty full proof.
+
+Before we get started, though, we need to understand the overall process here since each of the installation procedures is aiming at the same goal. In order to complete our installation, we need to achieve the following:
+
+1. Install the database system itself.
+2. Ensure that the database server can run (automatically, in most cases).
+3. Create a user to match your default user account so that you can access the db directly from your own account.
+
+Note that the last step (#3), involves adding a new database user. Now you see more clearly perhaps what makes Postgres different than SQLite. We need to setup and manage different users in the database. When you install the database, unless something has gone wrong, you a base user with broad privileges is automatically created. This user is almost always called `postgres`. What we will do in Step #3 above is create an additional user with broad privileges that corresponds to your main user account on your operating system.
+
+Okay, let's get started. There are instructions here for Ubuntu and Mac:
```

Comment:
I think it would be totally fair at this point in the curriculum to recommend that Windows users set up a virtual environment (no need to walk them through it, we can just mention it) in VirtualBox or VMWare and follow the Ubuntu instructions.